pkg/portal/service/authz.go

```diff
@@ -54,18 +54,21 @@ func (s *AuthzService) AddAuthorizedUser(appID string, userID string) error {
 	return s.Collaborators.CreateCollaborator(c)
 }
 
-func (s *AuthzService) CheckAccessOfViewer(appID string) error {
+func (s *AuthzService) CheckAccessOfViewer(appID string) (userID string, err error) {
```

Comment:
It seems the returned `userID` is not used everywhere, is there any reason to return it?

Reply:
It may be used in https://github.com/authgear/authgear-server/issues/527